versions/1.0.0.md

```diff
@@ -238,14 +238,15 @@ version: 1.0.1
 
 Describes a source description (such as an OpenAPI description) that will be referenced by one or more workflows described within a Workflows Description.
 
-An object storing a map between named description keys and location URLs to the source descriptions (such as an OpenAPI description) this workflow SHALL apply to. Each source location `string` MUST be in the form of a URL and MAY be defined using relative references as defined by [RFC3986](https://spec.openapis.org/oas/latest.html#bib-RFC3986).
+An object storing a map between named description keys and location URLs to the source descriptions (such as an OpenAPI description) this workflow SHALL apply to. Each source location `string` MUST be in the form of a URL. 
+If a relative reference is used it MUST be in the form of a URI-reference as defined by [RFC3986 section 4.1](https://datatracker.ietf.org/doc/html/rfc3986#section-4.2).

```

Comment:
With apologies, I'm going to have to be pedantic here:  "URI-reference" (§4.1) includes both "URI" (with a scheme; §1.1.1) and "relative reference" (§4.2).  Also, you mention 4.1 in the text and 4.2 in the link.

Really, you can just say that the value is a URI-reference (§4.2) and that covers everything.

Reply:
@ndenny could you make the little update suggested?